assets/check

```diff
@@ -34,7 +34,7 @@ configure_git_global "${git_config_payload}"
 destination=$TMPDIR/git-resource-repo-cache
 
 tagflag=""
-if [ -d $tag_filter ]; then
```

Comment:
Bleh, that looks broken, thanks for noticing. I'm gonna fix that on `master` just to unbreak things independent of this PR. I wonder why the tests didn't catch this?

Reply:
well this condition is only used to add a new `--tags flag` on git (the magic to detect tags on different branch). But the test are only done on one branch (branch-a) so it have no impact on the result. I not check how it could be done but the idea would be

`make a commit + tag on branch-b, clone --single-branch branch-a and look at tags on branch-b`

Reply:
My bad! I'll look into writing some tests to cover this

Reply:
Im not sure how you will able to cover it with local branch creation because --tags is used to get the remote branch from a clone --single-branch. But localy you will already have all the branchs.

Well hope you will find a tricks :)